x-pack/solutions/observability/packages/kbn-scout-oblt/src/playwright/fixtures/worker/profiling/api_supertest.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { formatRequest } from '@kbn/server-route-repository';
+import type request from 'superagent';
+import type supertest from 'supertest';
+import { format } from 'url';
+
+export function createProfilingApiClient(st: supertest.Agent) {
```

Comment:
It doesn't look like this function does anything specific about Profiling, does it? Should we create a more generic function if yes?

Reply:
I'll refactor this when I'm back